src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Col,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import CreatePasswordModal from './Modals/CreatePasswordModal';
+import ResetPasswordModal from './Modals/ResetPasswordModal';
+
+import css from './CreateResetPasswordControl.css';
+
+class CreateResetPasswordControl extends React.Component {
+  static propTypes = {
+    email: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    userId: PropTypes.string.isRequired,
+    resources: PropTypes.shape({
+      isLocalPasswordSet: PropTypes.shape({
+        records: PropTypes.object,
+      }).isRequired,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      resetPassword: PropTypes.shape({
```

Comment:
Please, add `isLocalPasswordSet` to propTypes

Reply:
We do not need this variable for mutator. And also we do not have it also in response